splunkapp/Docker/build.sh

```diff
@@ -4,7 +4,10 @@ build_package() {
     
     cd /pkg
     tar -zcvf ${wazuh_splunk_pkg_name} SplunkAppForWazuh
-    mv ${wazuh_splunk_pkg_name} ../wazuh_splunk_app
+    if [[ $4 == "yes"]]; then
+         sha512sum "${wazuh_splunk_pkg_name}" > "${wazuh_splunk_pkg_name}".sha512
+    fi
+    mv ${wazuh_splunk_pkg_name}.* ../wazuh_splunk_app
     cd ..
 }
 if [ $3 == "" ]; then
```

Comment:
Same as before.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/ba5843f8aac9201fe518b0d257c88d6493431ec2 and https://github.com/wazuh/wazuh-packages/pull/192/commits/5de766ec4dd5cddd21718c5897f9431117361be8

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/0564c0490c4129407572f7cf595ff898470d26bf